admin/app/com/lucidchart/piezo/admin/controllers/TriggerFormHelper.scala

```diff
@@ -91,7 +92,7 @@ class TriggerFormHelper(scheduler: Scheduler, monitoringTeams: MonitoringTeams)
     ),
   ] = {
     val trigger = value.trigger
-    val (triggerType: String, simple, cron) = trigger match {
+    val (triggerType: String, simple, cron) = (trigger: @unchecked) match {
```

Comment:
nit: I'd prefer adding a default case (`case other => throw new MatchError(other)`) rather than annotating it as unchecked